pkg/runtime/cli.go

```diff
@@ -114,9 +115,9 @@ func FromFlags() (*DaprRuntime, error) {
 		}
 	}
 
-	placementAddress := ""
+	placementAddress := []string{}
 	if *placementServiceHostAddress != "" {
-		placementAddress = *placementServiceHostAddress
+		placementAddress = parsePlacementAddr(*placementServiceHostAddress)
```

Comment:
For placementServiceHostAddress, change help message for flag.

Reply:
corrected.